languages/python/exercises/concept/dicts/.docs/introduction.md

```diff
@@ -0,0 +1,75 @@
+A _dictionary_ is Python's primary mapping type that connects _hashable keys_ with values. The looking up of keys is more efficient than searching through an array, but does require more memory. For a detailed explaination of _hashmaps_ visit [this][hashmap-blog] blog post.
```

Comment:
Reiterating on @ErikSchierboom 's suggestion (https://github.com/exercism/v3/pull/1936#discussion_r454289876):
> The introduction.md should preferrably not link to external resources (see the docs). The resource you're linking to though is an excellent one for the hints.md and after.md files.

```suggestion
A _dictionary_ is Python's primary mapping type that connects _hashable keys_ with values. The looking up of keys is more efficient than searching through an array, but does require more memory.
```

Reply:
I removed the piece from `introduction.md`, later I will add another piece about hashing to àfter.md`. I don't think we should add it to `hints.md` because it is not that relevant to the exercise (That's why we removed from the previous file anyway 😄). What do you think?

Reply:
I agree - adding it to hints would be a distraction.  In `after.md`, (_since by then the student will be comfortable with `dicts`_) you can add in both the link and some extra information/context around hashmaps & hashing.

Reply:
Sounds good

Reply:
Okay!